sos/policies/distros/redhat.py

```diff
@@ -430,11 +433,26 @@ def upload_sftp(self):
                                                        password=_token)
         raise Exception("Could not retrieve valid or anonymous credentials")
 
+    def check_file_too_big(self, archive):
+        size = os.path.getsize(archive)
+        # Lets check if the size is bigger than the limit.
+        # There's really no need to transform the size to Gb,
+        # so we don't need to call any size converter implemented
+        # in tools.py
+        if len(str(size)) >= 10 and (size >= self._max_size_request):
```

Comment:
Why the first condition `len(str(size)) >= 10`? That implies from the second, no?

Reply:
yes, that's a leftover from the first implementation when I was transforming the size to "1GB" and overcomplicating things. I'll remove it.

Reply:
Fixed